staff_features/resources/resource_merge.feature

```diff
@@ -1,35 +1,34 @@
-Feature: Resource Merge
+Feature: Resource merge
   Background:
     Given an administrator user is logged in
-      And a Resource with name 'A' has been created
-      And a Resource with name 'B' has been created
-  Scenario: Merge two Resources by browsing Resource
-    Given the Resource 'A' is opened in edit mode
+      And two Resources have been created
+  Scenario: Merge two Resources by browsing
+    Given the first Resource is opened in edit mode
```

Comment:
I think it is better to use Resource A and Resource B as in this way the steps are simpler to follow and more direct